services/cover/src/service/cover.rs

```diff
@@ -58,25 +58,19 @@ pub fn update_progress(update_progress: UpdateProgress) -> Result<(), Error> {
         .map_err(|e| e.into())
 }
 
-pub fn add_verification(
+pub fn submit_verification(
     caller_id: CallerId,
```

Comment:
Please keep add and update verification apis - we might switch to them when we introduce a security layer

Reply:
so may I closed the PR @rbialek, since you don't need it anymore?

Reply:
I need submit_verification on top of add_verifcation and update_verification

Reply:
it's exactly the code is doing, isn't it?

Reply:
yes, but we loose the ability to call add update from the outside.